cmd/tsbs_generate_data/main.go

```diff
@@ -107,6 +108,7 @@ func validateGroups(groupID, totalGroups uint) (bool, error) {
 	return true, nil
 }
 
+// validateFormat checks whether provided data format (one of formatChoices) is valid
```

Comment:
`// validateFormate checks whether format is valid (i.e., one of formatChoices)`

Reply:
Agree